controllers/delete.go

```diff
@@ -62,3 +64,30 @@ func (r *StorageSystemReconciler) deleteResources(instance *odfv1alpha1.StorageS
 
 	return fmt.Errorf("Waiting for %s %s to be deleted", instance.Spec.Kind, instance.Spec.Name)
 }
+
+func (r *StorageSystemReconciler) deleteQuickStarts(logger logr.Logger) {
```

Comment:
I have moved the func to quickstart files. The reason i am not returning any errors here is because we don't want to fail the operator if the func fails. We just log it and move on with rest of the cleanup.

I have updated the func to check number of StorageSystems, if it is 1 or less, then quickstarts won't be deleted